fuel-core/src/model/fuel_block.rs

```diff
@@ -69,72 +67,84 @@ impl BlockHeight {
     }
 }
 
-#[derive(Clone, Debug, Default, Deserialize, Serialize, PartialEq)]
-pub struct TransactionCommitment {
-    pub sum: Word,
-    pub root: Bytes32,
-}
-
 #[derive(Clone, Debug, Deserialize, Serialize)]
-pub struct FuelBlockHeaders {
-    pub fuel_height: BlockHeight,
+pub struct FuelBlockHeader {
+    /// Fuel block height.
+    pub height: BlockHeight,
+    /// Ethereum block number.
+    pub number: BlockHeight,
+    /// Block header hash of the previous block.
+    pub prev_hash: Bytes32,
```

Comment:
usualy called `parent` or `parent_hash`, but still `prev_hash` is fine name

Reply:
I like parent hash as well